.github/workflows/submitBeta.yml

```diff
@@ -31,10 +31,16 @@ jobs:
           file: ./extension/public/static/manifest.json
           field: version_name
           value: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.version }}
-      - name: Update manifest.json version for Chrome
+      - name: Update manifest-v2.json version for Chrome
         uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
         with:
-          file: ./extension/public/static/manifest.json
+          file: ./extension/public/static/manifest/v2.json
+          field: version
+          value: ${{ github.event.inputs.release }}.${{ github.event.inputs.version }}
+      - name: Update manifest-v3.json version for Chrome
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest/v3.json
```

Comment:
this probably isn't necessary, but adding it for symmetry. It will make it easier to just delete the v2 step later